slow_odgi/README.md

```diff
@@ -0,0 +1,350 @@
+### Overview
+
+`slow-odgi` is a reference implementation of [`odgi`](https://github.com/pangenome/odgi). It is written purely in Python, with correctness and clarity as goals and speed as a non-goal. Think of it as a code-ey spec for `odgi` commands.
+
+### Testing
+
+To test `slow-odgi`, we treat `odgi` as an oracle and compare our outputs against theirs. We mostly test against a set of GFA graphs available in the `odgi` repository, and, in a few cases, supplement these with short hand-rolled GFA files of our own.
+
+To run these tests, you will need 
+1. `odgi`; see [here](https://github.com/pangenome/odgi). Our tests were run against a built-from-source copy of `odgi` (commit 34f006f).
+2. `turnt`; see [here](https://github.com/cucapra/turnt).
+
+With these in place, run `make test-slow-odgi`. The "oracle" files will be generated first, and this will toss up a large number of warnings which can all be ignored. Then the tests will begin to run, and the `ok`/`not-ok` signals there are actually of interest. 
+
+There are a few known points of divergence versus `odgi`:
+1. `flip` disgrees against graphs note5.gfa and flip4.gfa.
+2. `inject` disagrees against graphs DRB1-3123.gfa and chr6.C4.gfa.
+
+
+### Explanation of Commands
+
+The remainder of this document will explain, in some detail, the eleven commands that we have implemented. Below we sometimes elide graph information that is inconsequential to the explanation. Unless specified, this is meant to be read as "don't care" and not as absence. 
+
+
+#### `chop`
```

Comment:
What do you think about adding quick, one-sentence explanations here? I guess they would not be too different from what's in the current odgi docs. But starting out with an overview to just say something like this could help people get the idea: "Shortens segments to a given maximum length, while preserving the end-to-end sequence represented by each path." Then we can launch into the example.